components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -324,7 +324,7 @@ On the project list page, the button `Update Projects` should be enabled since a
 
 ![](/images/docs/update-projects-button.png)
 
-Updating a project begins an operation that applies all pending rule edits and then moves ingested resources into the correct projects according to those latest changes. An ingested resource is moved into a project if it matches at least one of the project's rules.
+[Updating a project] ({{< ref "iam-v2-api-reference.md#applying-rules" >}}) begins an operation that applies all pending rule edits and then moves ingested resources into the correct projects according to those latest changes. An ingested resource is moved into a project if it matches at least one of the project's rules.
```

Comment:
Would the Applying Rules section be relevant, or would the "iam-v2-api-reference/#updating-a-project-rule" just above Applying Rules be more relevant?

Reply:
Alternatively, would a line under Applying Rules in iam-v2-api-reference.md clarify why this specific link be relevant?

Reply:
I'm not sure, @mjingle. I don't think it would hurt to have it in both places (for your rule changes to take effect, you'll need to _link to apply rules_) and here. But this button is specifically tied to the `apply-rules` endpoint.

Reply:
i think we could prolly drop the link... idk though

Reply:
Mk. I'll drop it for now.